src/Uno.UWP/Devices/Midi/MidiNoteOffMessage.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using Uno.Devices.Midi.Internal;
+using Windows.Storage.Streams;
+
+namespace Windows.Devices.Midi
+{
+	/// <summary>
+	/// Represents a MIDI messages that specifies a MIDI note to turn off.
+	/// </summary>
+	public partial class MidiNoteOffMessage : IMidiMessage
+	{
+		private readonly InMemoryBuffer _buffer;
+
+		/// <summary>
+		/// Creates a new MidiNoteOffMessage object.
+		/// </summary>
+		/// <param name="channel">The channel from 0-15 that this message applies to.</param>
+		/// <param name="note">The note which is specified as a value from 0-127.</param>
+		/// <param name="velocity">The velocity which is specified as a value from 0-127.</param>
+		public MidiNoteOffMessage(byte channel, byte note, byte velocity)
+			: this(new byte[] {
+				(byte)((byte)MidiMessageType.NoteOff| channel),
+				note,
+				velocity
+			})
+		{
+		}
+
+		internal MidiNoteOffMessage(byte[] rawData)
+		{
+			if ((rawData[0] & (byte)MidiMessageType.NoteOff) == (byte)MidiMessageType.NoteOff)
+			{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove or fill this block of code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)